k8s/grafana/apptest/tester/tests/basic-suite.yaml

```diff
@@ -1,8 +1,22 @@
 actions:
-- name: dummy test
+- name: grafana listening
   bashTest:
     script: |-
-      exit 0
+      sleep 120
+      curl http://$APP_INSTANCE_NAME-grafana.$NAMESPACE.svc.cluster.local:3000
     expect:
-      exitCode:
-        equals: 0
+      stdout:
+        contains: 'login'
+- name: graphana login
+  bashTest:
+    script: |-
+      ADMIN_PASSWORD="$(kubectl get secret "${APP_INSTANCE_NAME}-grafana" \
```

Comment:
Please move these variables to the `tester.yaml.template` file. Example: https://github.com/GoogleCloudPlatform/click-to-deploy/blob/ac871843164cebccb08b02905f98050f8957e2ff/k8s/wordpress/apptest/deployer/manifest/tester.yaml.template#L30-L50